frontend/pages/SoftwarePage/SoftwareVulnerabilityDetailsPage/SoftwareVulnOSVersions/SwVulnOSTableConfig.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+import { Column } from "react-table";
+
+import { IVulnerabilityOSVersion } from "interfaces/operating_system";
+
+import LinkCell from "components/TableContainer/DataTable/LinkCell";
+
+import PATHS from "router/paths";
+import SoftwareIcon from "pages/SoftwarePage/components/icons/SoftwareIcon";
+import TextCell from "components/TableContainer/DataTable/TextCell";
+import ViewAllHostsLink from "components/ViewAllHostsLink";
+
+interface ICellProps {
+  row: {
+    original: IVulnerabilityOSVersion;
+  };
+}
+
+interface INumberCellProps extends ICellProps {
+  cell: {
+    value: number;
+  };
+}
+
+const generateColumnConfigs = (isPremiumTier: boolean): Column[] => {
+  const configs = [
+    {
+      Header: "Name",
+      disableSortBy: true,
+      accessor: "name_only",
+      Cell: ({ row }: ICellProps) => {
+        const { name, os_version_id, platform } = row.original;
+        return (
+          <LinkCell
+            path={PATHS.SOFTWARE_OS_DETAILS(os_version_id)}
+            value={
+              <>
+                <SoftwareIcon name={platform} />
+                <span className="os-version-name">{name}</span>
+              </>
+            }
+          />
+        );
+      },
+    },
+    {
+      Header: "Version",
+      disableSortBy: true,
+      accessor: "version",
+      Cell: ({ cell }: INumberCellProps) => <TextCell value={cell.value} />,
+    },
+    {
+      Header: () => (
+        <>
+          Resolved in <div className="resolved-suffix">version</div>
+        </>
+      ),
+      disableSortBy: true,
+      accessor: "resolved_in_version",
+      Cell: ({ cell }: INumberCellProps) => <TextCell value={cell.value} />,
+    },
+    {
+      Header: "Hosts",
+      disableSortBy: true,
```

Comment:
This also should be default sorted on the frontend? @rachaelshaw

Reply:
@jacobshandling just sort on frontend for now, maybe for the select of the API response, just in case API doesn't have it sorted